about.html

```diff
@@ -24,7 +24,34 @@
     </style>
   </head>
   <body>
+    <div class="mobile-dropdown hidden">
+      <div class="close-x">
+        <img src="./assets/img/close-x.png" alt="close dropdown" width="80" />
+      </div>
+      <div class="menu-list">
+        <ul>
+          <li><a href="index.html">Home</a></li>
```

Comment:
- [ ] good job adding the home button as the previous reviewer requested, but it's only visible for the mobile menu, kindly add it to desktop navbar as well